### ISSUE TITLE
GCC version fix

### ISSUE THREAD

**Issue by lyssdod** [cmake-js/boost-lib#4]
While building example repo, got another error (after #3 ):

```
[30/42] Performing build step for 'boost_coroutine_jam'
FAILED: cd /home/lyssdod/.cmake-js/boost/1.61.0 && /usr/bin/cmake -P /storage/work/cmake-js-tut-04-boost-module/build/boost-1.61.0/boost_coroutine_jam-build-Release.cmake && /usr/bin/cmake -E touch /storage/work/cmake-js-tut-04-boost-module/build/boost-1.61.0/boost_coroutine_jam-build
CMake Error at /storage/work/cmake-js-tut-04-boost-module/build/boost-1.61.0/boost_coroutine_jam-build-Release.cmake:16 (message):
  Command failed: 1

   './b2' 'link=static' 'threading=multi' 'runtime-link=shared' '--build-dir=Build' '--ignore-site-config' 'stage' '--stagedir=stage' '-d+2' '--hash' 'variant=release' '--layout=tagged' '-sNO_BZIP2=1' 'cxxflags=-fPIC' 'cxxflags=-std=c++11' 'toolset=gcc-5.4' '--with-coroutine'

  See also

    /storage/work/cmake-js-tut-04-boost-module/build/boost-1.61.0/boost_coroutine_jam-build-*.log
```

Solution: https://safenetforum.org/t/build-on-arch-x64-fails/2209/2

**Comment by unbornchikken** [comment]
Does https://github.com/cmake-js/boost-lib/pull/6 resolve this?

**Comment by lyssdod** [comment]
No. I will open a PR if you don't mind

**Comment by lyssdod** [comment]
Fixed by #7
